usaspending_api/download/migrations/0004_download_views.py

```diff
@@ -12,7 +12,9 @@ class Migration(migrations.Migration):
         ("accounts", "0002_auto_20200320_1307"),
         ("awards", "0074_financialaccountsbyawards_distinct_award_key"),
         ("download", "0003_auto_20180306_1726"),
-        ("financial_activities", "0003_financialaccountsbyprogramactivityobjectclass_disaster_emergency_fund")
+        ("financial_activities", "0003_financialaccountsbyprogramactivityobjectclass_disaster_emergency_fund"),
```

Comment:
Why is this migration being updated instead of adding a new one? Won't this migration already be marked as having run?

Reply:
Correct. Realistically the migration is put in place for the sake of tests and the model. The actual SQL run to create the views can be run manually. For example, if in the future we need to add another field to the SQL View then it wouldn't make sense to add another migration. Instead we would add a field to the model, the View, and then re-run the SQL that DROPs and CREATEs the View.

Benefit of this approach is that all Download SQL View migrations are in one place.

Reply:
So will this ticket include a Data Change ticket to run that SQL?

Reply:
Sure will. Still need to make that, but will include it as a step.